R/eval_forecasts.R

```diff
@@ -238,16 +279,47 @@ eval_forecasts <- function(data,
 
     if (summarised) {
 
-      res <- res[, .("interval_score" = mean(interval_score),
-                     "calibration" = ifelse(exists("calibration"), mean(calibration), NA),
-                     "bias" = ifelse(exists("bias"), mean(bias), NA),
-                     "sharpness" = ifelse(exists("sharpness"), mean(sharpness), NA)),
-                 by = c(by, "range")]
+      if (!is.null(quantiles)) {
```

Comment:
This is absolutely fine but it might be cleaner to operate on everything in long format (so metric, value) and return summaries then move back to wide rather than repeating code. If not going to be adding new measures (or just a few) this really doesn't matter and may be a lot more overhead than its worth.